rest_framework/schemas/generators.py

```diff
@@ -150,8 +150,8 @@ def should_include_endpoint(self, path, callback):
             return False  # Ignore anything except REST framework views.
 
         if hasattr(callback.cls, 'exclude_from_schema'):
-            fmt = ("{}. The `APIView.exclude_from_schema` is pending deprecation. "
-                   "Set `schema = None` instead")
+            fmt = ("The `APIView.exclude_from_schema` is pending deprecation. "
+                   "Set `schema = None` instead in `{}`.")
```

Comment:
Just before I merge this... 

I put the class name first so it would be more obvious in the console. 

```
Last it could be all the way ..................................................................................................................................................................................................................over here.
```

What do you think?

Reply:
@rpkilby PING 🙂

Reply:
That's fair. Maybe `APIView.exclude_from_schema` => `ClassName.exclude_from_schema`?

Reply:
I mostly made the change b/c the class name on its own read a little oddly.

Reply:
OK. Do that and we'll go. 👍

Reply:
Updated.